q2cli/util.py

```diff
@@ -178,3 +182,23 @@ def callback(ctx, param, value):
     return click.Option(['--citations'], is_flag=True, expose_value=False,
                         is_eager=True, callback=callback,
                         help='Show citations and exit.')
+
+
+def usage_example_option(action):
```

Comment:
It would be really cool to have an `--example-data ./foo_dir` command, that would render out the usage data. I'm not so sure what the best way is to do this, though, since we won't have references to the data anymore. Options include:

- make a one-off `ExecutionUsage`-style usage driver, that gets invoked directly when rendering the data - this would allow us to bypass the cache
- save the example data in the cache when we initialize the cache. that sounds like it will backfire, but I am open to it.

Please leave this until the end - this is more of a "nice-to-have" than a "must-have" for getting this feature merged.

Reply:
Let's save this for 2020.8, I looked at what we would need to do to implement this, and I am not comfortable adding that much functionality this close to the release.